test/haggadah/dsl_test.cljs

```diff
@@ -2,79 +2,45 @@
-     [:tr  
-      [:td "Piojos"]
-      [:td "כִּנִּים"]]
-     [:tr
-      [:td "Bestias"]
-      [:td "עָרוֹב"]]
-     [:tr  
-      [:td "Peste"]
-      [:td "דֶּבֶר"]]]]])
-
-(def table-content
-  [["Sangre" "דָּם"]
-   ["Ranas" "צְפַרְדֵּעַ"]
-   ["Piojos"  "כִּנִּים"]
-   ["Bestias"  "עָרוֹב"]
-   ["Peste" "דֶּבֶר"]])
-
-(def table-title "Las Diez Plagas")
-
-(t/deftest haggadah-with-table-test
-  (t/testing "When the user creates a Haggadah with a table in it and the Haggadah is parsed, the correct hiccup representation of the Haggadah is returned"
-    (let [haggadah (dsl/create-haggadah-with-table table-title table-content)
-          hiccup-rep (dsl/parse-haggadah (:content haggadah))]
-      (t/is (= haggadah-with-table hiccup-rep)))))
-
-(def subsection-title "Subsection")
-
-(def haggadah-with-subsection
-  [:div
-   [:div.has-text-centered.has-text-weight-bold.is-size-3.pb-2 subsection-title]
-   haggadah-as-hiccup
-   haggadah-with-table])
-
-(def subsection-content
-  [{:song {:title title :content song}}
-   {:table {:title table-title :content table-content}}])
-
-
-(t/deftest haggadah-with-subsection-test
-  (t/testing "When the user creates a Haggadah with a subsection and the Haggadah is parsed, the correct hiccup representation of the Haggadah is returned"
-    (let [haggadah (dsl/create-haggadah-with-subsection subsection-title subsection-content)
-          hiccup-rep (dsl/parse-haggadah (:content haggadah))]
-      (t/is (= haggadah-with-subsection hiccup-rep)))))
+(def bracha (dsl/bracha "Wine" "Baruj hata"))
+(def expected-bracha
+  [:div.bracha
+   [:div.title  "Wine"]
+   [:div.text  "Baruj hata"]])
+
+(t/deftest render-bracha-test
+  (t/testing "When rendering a bracha, returns the title and the text"
+    (t/is (= expected-bracha (dsl/render-haggadah bracha)))))
+
+
+(t/deftest render-song-test
+  (t/testing "When rendering the song, returns the title and content"
+    (let [song (dsl/song "Ki lo nae" "כִּי לוֹ נָאֶה, כִּי לוֹ יָאֶה. אַדִּיר בִּמְלוּכָה, בָּחוּר כַּהֲלָכָה, גְּדוּדָיו")
+          expected [:div.song
+                    [:div.title "Ki lo nae" ]
+                    [:div.text "כִּי לוֹ נָאֶה, כִּי לוֹ יָאֶה. אַדִּיר בִּמְלוּכָה, בָּחוּר כַּהֲלָכָה, גְּדוּדָיו"]]]
+      (t/is (= expected (dsl/render-haggadah song))))))
+
+(t/deftest render-table-test
+  (t/testing "When rendering the table, each row and the content within is returned"
+    (let [table (dsl/table "Las Diez Plagas" 
+                           (dsl/row ["Sangre" "דָּם"])
+                           (dsl/row ["Ranas" "צְפַרְדֵּעַ"]))
+          expected [:div.table.is-bordered
+                    [:div.title "Las Diez Plagas"]
+                    [:table.table.table-content
+                     [:tbody
+                      [:tr [:td "Sangre"] [:td "דָּם"]]
+                      [:tr [:td "Ranas"] [:td "צְפַרְדֵּעַ"]]]]]]
+      (t/is (= expected (dsl/render-haggadah table))))))
+
+(def expected-section
```

Comment:
NIT: Maybe using the `def` inside the test doesn't look bad

Reply:
I'll add the def inside the test to see how it looks in comparison.